frontend-react/.env.test

```diff
@@ -1,3 +1,20 @@
-REACT_APP_BACKEND_URL=http://localhost:8090
+;; this is set by env var being set `CLIENT_ENV: test` in the github action
+REACT_APP_TITLE=CDC Prime ReportStream TEST
+REACT_APP_DESCRIPTION=ReportStream Dashboard Application
+REACT_APP_ICON=https://reportstream.cdc.gov/assets/cdc-logo.svg
+
+REACT_APP_BASE_URL=http://localhost:3000
+REACT_APP_BACKEND_URL=http://localhost:7071/api/
+; REACT_APP_BACKEND_URL=http://localhost:8090 ????
```

Comment:
8090?? Odd. Does this impact development at all? I'm wondering if `REACT_APP_BACKEND_URL` should be `localhost:3000` here too like it is in `.env.development`.

Reply:
That line is commented out, but I agree it's odd. The old value was 8090, so I left it around in case something broke, but I don't think "test" is a real thing yet.

Reply:
I think Test is just building the project after it merges and this doesn't seem to break that build, so I'm cool keeping it commented out!